assets/js/modules/adsense/components/module/ModuleTopEarningPagesWidget/index.js

```diff
@@ -70,15 +71,33 @@ function ModuleTopEarningPagesWidget( {
 				limit: 10,
 			};
 
+			const report = select( MODULES_ANALYTICS ).getReport( reportArgs );
+
+			const pageTitles = select( MODULES_ANALYTICS ).getPageTitles(
+				report,
+				{
+					startDate,
+					endDate,
+				}
+			);
```

Comment:
See above, I suggest we simplify this to:
```suggestion
			const pageTitles = select( MODULES_ANALYTICS ).getPageTitles( report, reportArgs );
```